libsolidity/formal/CHC.cpp

```diff
@@ -1268,7 +1234,8 @@ void CHC::checkAssertTarget(ASTNode const* _scope, CHCVerificationTarget const&
 	{
 		auto it = m_errorIds.find(assertion->id());
 		solAssert(it != m_errorIds.end(), "");
```

Comment:
I just noticed this, but actually these two lines could be moved outside the inner loop (before the loop).
After that I think it's good to merge.

Reply:
We are looping over a set of assertions and we are looking for the error ids for each individual assertion. I don't see how you would like to move that outside of the loop.

Reply:
Ah sorry, it wasn't here, got confused in the diff, going to comment on the one I meant